docs/docs/how-to/images-and-media/using-gatsby-plugin-image.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Using the beta Gatsby Image plugin
+---
+
+Adding responsive images to your site while maintaining high performance scores can be difficult to do manually. The Gatsby Image plugin handles the hard parts of producing images in multiple sizes and formats for you!
+
+> Want to learn more about image optimization challenges? Read the Conceptual Guide: [title of conceptual guide, hyperlinked].
+
+The new Gatsby Image plugin is currently in beta, but you can try it out now and see what it can do for the performance of your site.
+
+## Getting started
+
+First you need to install the following packages:
+
+```shell
+npm install gatsby-plugin-image gatsby-plugin-sharp gatsby-transformer-sharp
+```
+
+You then need to add the plugins to your `gatsby-config.js`:
+
+```js:title=gatsby-config.js
+module.exports = {
+  plugins: [
+    `gatsby-plugin-image`,
+    `gatsby-plugin-sharp`,
+    `gatsby-transformer-sharp`,
+  ],
+}
+```
+
+If you already have some of these plugins installed, please check that they're updated to the latest version.
+
+<!-- TODO: add exact minimum version when we reach GA -->
+
+## Using the Gatsby Image components
+
+### Decide which component to use
+
+The Gatsby Image plugin includes two image components: one for static and one for dynamic images. The simplest way to decide which you need to is to ask yourself: _"will this image be the same every time the component or template is used?"_. If it will always be the same, then use `StaticImage`. If it will change, whether through data coming from a CMS or different values passed to a component each time you use it, then it is a dynamic image and you should use the `GatsbyImage` component.
+
+### Static images
+
+If you are using an image that will be the same each time the component is used, such as a logo or front page hero image, you can use the `StaticImage` component. The image can be either a local file in your project, or an image hosted on a remote server. Any remote images are downloaded and resized at build time.
+
+1. **Add the image to your project.**
+
+   If you are using a local image, copy it into the project. A folder such as `src/images` is a good choice.
```

Comment:
You need `gatsby-source-filesystem` for static image, so we should include that here.

Reply:
Good catch. Maybe it make sense to add that as a prerequisite in the Getting Started section?

I'm wondering if it makes sense to add a note telling them they'll need `gatsby-source-filesystem` and then link them away to the how-to guide for adding that plugin. Or if we should just add the steps directly in this doc (which might be a bit off topic but at least saves them from jumping between docs). Thoughts?

Reply:
The caveat is you don't need that plugin if you're using a CMS with GatsbyImage. It's only necessary for StaticImage because it uses the createFileNode API. So since this doc covers both I wouldn't think it should be a prerequisite.

Reply:
Got it. You're right, it makes the most sense to add installing `gatsby-source-filesystem` as a step in the StaticImage section. Thanks for explaining it! :)